documentation/api/query/v4/operators.markdown

```diff
@@ -299,19 +299,57 @@ An `in` statement constitutes a full query string, which can be used alone or as
 
 "In" statements are **non-transitive** and take two arguments:
 
-* The first argument **must** consist of one or more **fields** for the endpoint or entity **being queried.**. This is a string or vector of strings.
+* The first argument **must** consist of one or more **fields** for the endpoint
+  or entity **being queried.**. This is a string or vector of strings.
 * The second argument **must** be either:
-** an **`extract` statement,** which acts as a list of fields to extract during the subquery for matching against the **fields** in the `in` clause.
-** a **`from` statement,** which sets the context, and allows for an extract statement to be provided. *Note:* this syntax is new and experimental.
+** an **`extract` statement,** which acts as a list of fields to extract during
+   the subquery for matching against the **fields** in the `in` clause.
+** a **`from` statement,** which sets the context, and allows for an extract
+   statement to be provided. *Note:* this syntax is new and experimental.
 
 **Matches if:** the field values are included in the list of values created by the `extract` or `from` statement.
 
+##### `array`
+
+An `in` statement also accepts an `array` statement as a second argument.
```

Comment:
the section for "in" above this contains some must language around the second argument that needs to be updated. Doing that may make this redundant.